contracts/modules/IdleLiquidityModule.sol

```diff
@@ -36,13 +37,13 @@ contract IdleLiquidityModule is BaseModule {
         uint24 tickDistance = _checkDistanceFromRange(tokenId);
         (, bytes32 rebalanceDistance) = positionManager.getModuleInfo(tokenId, address(this));
 
-        ///@dev rebalance only if the position's range is outside of the tick of the pool (tickDistance < 0) and the position is far enough from tick of the pool
-        if (tickDistance > 0 && uint24(uint256(rebalanceDistance)) <= tickDistance) {
+        ///@dev can rebalance only if the position range is outside of the pool tick and it is far enough from the pool tick
```

Comment:
It's the inverse: can rebalance only if the pool tick is outside the position range